dds_cli/__main__.py

```diff
@@ -490,6 +490,100 @@ def get_info_user(click_ctx, username):
         sys.exit(1)
 
 
+# -- dds user activate -- #
+@user_group_command.command(name="activate", no_args_is_help=True)
+# Positional args
+@email_arg(required=True)
+# Options
+@username_option()
+# Flags
+@click.pass_obj
+def activate_user(click_ctx, email, username):
+    """
+    (Re)Activate user accounts in the Data Delivery System.
+
+    If you have sufficient admin privileges, you may activate the accounts of other users.
+    Specify the e-mail address as argument to the main command to initiate the activation process.
+    """
+    if click_ctx.get("NO_PROMPT", False):
+        pass
+    else:
+        proceed_activation = rich.prompt.Confirm.ask(
+            f"Activate Data Delivery System user account associated with {email}?"
+        )
+
+    if proceed_activation:
+        try:
+            with dds_cli.account_manager.AccountManager(
+                username=username,
+                no_prompt=click_ctx.get("NO_PROMPT", False),
+            ) as manager:
+                if email:
+                    manager.user_activation(email=email, action="reactivate")
+                else:
+                    LOG.error(
+                        "You must either specify the the e-mail "
+                        "address of the user to be deleted"
+                    )
+                    sys.exit(1)
```

Comment:
Not needed since the `email_arg` is required.

Reply:
Done!